scan/git.go

```diff
@@ -30,7 +30,8 @@ type gitRepo struct {
 func (s *Scanner) GetGitCommitID(ctx context.Context, cmdDir string) (string, error) {
 	cmd := []string{"git", "rev-parse", "--verify", "HEAD"}
 	var buf bytes.Buffer
-	if err := s.procManager.Run(ctx, cmd, nil, &buf, os.Stderr, cmdDir); err != nil {
+	if err := s.procManager.Run(ctx, cmd, nil, &buf, nil, cmdDir); err != nil {
+		log.Printf("WARNING: cannot query commit: %s\n", err)
```

Comment:
log.Printf will be output to end user and the warning message will be confusing. i think you can just remove it.